src/dom/DomEvent.Pointer.js

```diff
@@ -31,7 +31,7 @@ export function addPointerListener(obj, type, handler) {
 	if (type === 'touchstart') {
 		_addPointerDocListener();
 	}
-	handler = handle[type].bind(this, handler);
+	handler = handler[type].bind(this, handler);
```

Comment:
Are you sure that this is correct? Because the `handle` object contains the mapping to the functions.

https://github.com/Leaflet/Leaflet/blob/99361e805654d01e4adf4e325e677a2870aa16cd/src/dom/DomEvent.Pointer.js#L18-L23